tests/test_bgp_af.rb

```diff
@@ -1019,35 +810,37 @@ def table_map(asn, vrf, af)
     #
     val = 'sjc'
     bgp_af.table_map_set(val)
-    assert_equal(val, bgp_af.table_map,
-                 'Error: default metric value does not match set value')
+    assert_equal(val, bgp_af.table_map)
```

Comment:
Basically, all the assert messages were just typing out the same information that gets printed out when the assert fails.  This seems wasteful to me.  In some cases, the failing-assert message was incorrect and I think the original author added a specific message to correct the auto-generated messages.  Actually, it just needed the order of the parameters to be swapped to make the auto-generated message correct.  I've done this for all the asserts I've come across so far, and I prefer the auto-generated messaging since it can be internationalized more easily.